### PR TITLE
Refactor mui button

### PR DESCRIPTION
**Pull request by Lumiere-Lodi**
This PR refactor styling of button component across the entire application.

### CODE REVIEW

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: src/components/reviewMatches/MatchDetails.tsx [hunk 1]
```diff
@@ -377,14 +377,18 @@ const MatchDetails = () => {
...
-              <CircularProgress />
-            ) : (
-              'Confirm'
-            )}
-          </Button>
+          <CustomButton onClick={handleCancel} label="Cancel" />
+          <CustomButton
+            onClick={handleConfirm}
+            autoFocus
+            label={
```
Comment (marrouchi) [inline src/components/reviewMatches/MatchDetails.tsx:384]: IsLoading should be added as a boolean prop.

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: src/components/auditTrail/AuditTrail.tsx [hunk 2]
```diff
@@ -121,17 +122,11 @@ const AuditTrail = () => {
...
         </Grid>
         <Grid item>
-          <Button
-            variant="contained"
-            sx={{
-              height: '36px',
-              width: '152px',
-              borderColor: theme => theme.palette.primary.main
-            }}
+          <CustomButton
```
[the remark this answers is not in the document]
Comment (Lumiere-Lodi) [reply src/components/auditTrail/AuditTrail.tsx:125]: noted.

File: src/components/auditTrail/AuditTrail.tsx [hunk 3]
```diff
@@ -121,17 +122,11 @@ const AuditTrail = () => {
...
+            variant="headerButton"
             href={`patient/${uid}`}
-          >
-            <Typography variant="button">BACK TO RECORD</Typography>
-          </Button>
+            label={<Typography variant="button">BACK TO RECORD</Typography>}
```
[the remark this answers is not in the document]
Comment (Lumiere-Lodi) [reply src/components/auditTrail/AuditTrail.tsx:128]: I think I didn't refactor some of the existing codes. I will have a look at that. This is part of the main. I will refactor it. It shouldn't be this way, I think.

File: src/components/customSearch/CustomSearch.tsx [hunk 4]
```diff
@@ -73,21 +74,8 @@ const CustomSearch: React.FC = () => {
...
-                  key="simple-search"
-                  variant="outlined"
-                  sx={{
-                    height: '42px',
-                    width: '172px',
-                    borderColor: theme => theme.palette.primary.main
-                  }}
-                  href={'/search/simple'}
-                >
+                <SearchFlags options={options} onChange={setIsGoldenOnly} />,
```
[the remark this answers is not in the document]
Comment (Lumiere-Lodi) [reply src/components/customSearch/CustomSearch.tsx:77]: sure.

File: src/components/reviewMatches/MatchDetails.tsx (hunk 1 above)
Comment (Lumiere-Lodi) [reply src/components/reviewMatches/MatchDetails.tsx:384]: sure.

**Review by MatthewErispe — APPROVED** [review]
@Lumiere-Lodi LGTM

**Review by marrouchi — APPROVED** [review]
@Lumiere-Lodi Awesome work :clap:

**Pull request merged**